mmr-backend/book.py

```diff
@@ -14,16 +14,16 @@ class Book:
     publishing_date: str
     edition: str
 
-    def fromList(books):
+    def from_list(books):
         bookList: list[Book] = []
         for book in books:
-            bookList.append(Book.fromDict(book))
+            bookList.append(Book.from_dict(book))
         return bookList
 
-    def fromDict(book):
+    def from_dict(book):
```

Comment:
You're repeating literals a bit too much. You could put them in a constant list, and reuse them through a loop.

Reply:
I'll take a look for some possible solution, because I don't really like updating that list each time I edit/add attributes (which is the same problem I have right now)

Maybe there's some way to get all attributes from an instance...

Reply:
Introspection is your friend.